src/styles/bootstrap-theme/colors.module.scss

```diff
@@ -27,18 +27,31 @@ $dark: $slate;
 $light: $platinum;
 $info: $sea-green;
 
-$custom-colors: (
-  'white': $white,
+$theme-colors: (
+  'primary': $primary,
+  'secondary': $secondary,
+  'success': $success,
+  'info': $info,
+  // 'warning': $meyer-lemon,
```

Comment:
The original theme colors in `bootstrapVariables` included 'warning'. Should `$meyer-lemon` be used for warning, or should I put the default bootstrap's hex for 'warning?'

Reply:
We still want the default warning, see previous comment. meyer-lemon is a separate color. No need to declare the warning color here, see previous comment.

Reply:
Also we still need `meyer-lemon` to be declared here, since it's used in at least 1 place I know about.